charts/posthog/templates/_snippet-initContainers-wait-for-migrations.tpl

```diff
@@ -0,0 +1,51 @@
+{{/* Common initContainers-wait-for-migrations definition */}}
+{{- define "_snippet-initContainers-wait-for-migrations" }}
+- name: wait-for-migrations
+  image: {{ include "posthog.image.fullPath" . }}
+  command:
+    - /bin/sh
+    - -c
+    - >
+        until (python manage.py migrate --check);
+        do
+            echo "Waiting for database migrations to be completed"; sleep 1;
+        done
+  env:
+
+  # PostgreSQL configuration
```

Comment:
this isn't super intuitive that it's in a file named `wait-for-migrations`

Reply:
PostgreSQL env variables?